api/services/service-auth-provider-api/service-auth-provider-api.spec.ts

```diff
@@ -3,20 +3,18 @@ import * as chai from 'chai'
 import * as sinon from 'sinon'
 import * as sinonChai from 'sinon-chai'
 import * as express from 'express'
-import * as pq from 'proxyquire'
 import * as supertest from 'supertest'
 
 chai.use(sinonChai)
 const expect = chai.expect
 const assert = chai.assert
-const proxyquire = pq.noPreserveCache()
 
 import {config} from '../../../config'
```

Comment:
missing alot of semicolons on the test, smh :joy:

Reply:
yeah, it's going to be a long process